alphas/alpha_functions.py

```diff
@@ -10,22 +10,78 @@
 from __future__ import annotations
 
 import pandas as pd
+from quant_portfolio_tool.alphas.alpha_utils import *
```

Comment:
This will be a problem for other people who don't have the same setup as you. I figured the easiest way to do this is by doing what I did in the other alpha files, using the `os`, 'sys`, and `pathlib` libraries. That way it works no matter who runs it